src/inmanta/graphql/schema.py

```diff
@@ -29,25 +29,35 @@
 
 
 def get_expert_mode(root: "Environment") -> bool:
-    if not hasattr(root, "settings"):
-        return False
+    assert hasattr(root, "settings")
     return bool(root.settings.get("enable_lsm_expert_mode", False))
 
 
-@mapper.type(models.Project)
-class Project:
-    pass
-
-
 @mapper.type(models.Environment)
 class Environment:
+    # Add every relation/attribute that we don't want to expose in our GraphQL endpoint to `__exclude__`
+    __exclude__ = [
+        "project_",
+        "agentprocess",
+        "code",
+        "compile",
+        "configurationmodel",
+        "discoveredresource",
+        "environmentmetricsgauge",
+        "environmentmetricstimer",
+        "notification",
+        "parameter",
+        "resource_persistent_state",
+        "unknownparameter",
+        "agent",
+    ]
```

Comment:
This feels a bit weird to exclude both "specific" stuff (the `project_` relation on the environment table) and "global"/'top level" things such as tables in here.
If we ever add another mapper would we have to exclude the tables we don't want again ?

Reply:
I'm not sure I follow and what the difference is between the "global" and "specific" stuff.
I excluded every relationship coming out of  `Environment` so that we don't expose more than we want to to the graphql API.
If we add another strawberry class that is mapped to an SQLAlchemy table, we would have to exclude every relation/attribute that we don't want as well

Reply:
You can Ignore this comment, I didn't realize those were actual relation names (I though they were referring to the table themselves)